### PR TITLE
setup local development environment

### PR DESCRIPTION
**Pull request by abrhamgg**

### CODE REVIEW

File: 2022-round-2/abrhamgg/nx_dev_test_output.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+PYTHONPATH=. pytest networkx
```
Comment (MridulS) [inline 2022-round-2/abrhamgg/nx_dev_test_output.txt:1]: Could you paste the output of the test suite?
Thanks!

**Comment by abrhamgg** [comment]
completed second task

File: 2022-round-2/abrhamgg/nx_dev_test_output.txt (hunk 1 above)
Comment (abrhamgg) [reply 2022-round-2/abrhamgg/nx_dev_test_output.txt]: done

**Comment by MridulS** [comment]
Great! Thanks @abrhamgg

**Pull request merged**